### ISSUE TITLE
Cannot create new project in Ubuntu 16.04

### ISSUE THREAD

**Issue by vt45** [kunstmusik/blue#346]
with Openjdk "1.8.0_111".
The applications starts but the item "Create New Project" does not respond. See attachment.
Regards,
Martin
[ubuntu16.04.txt](https://github.com/kunstmusik/blue/files/673348/ubuntu16.04.txt)

**Comment by kunstmusik** [comment]
This looks like a problem with missing JavaFX. I think on Linux, when using OpenJDK you need to install javafx as a separate package.  (If you use Oracle JDK, it should come with JavaFX.)  Could you check to see if you have JavaFX installed?

**Comment by vt45** [comment]
I installed openjfx but didn't solve the issue; I've attached a new log file.

[blue2.txt](https://github.com/kunstmusik/blue/files/674961/blue2.txt)

**Comment by kunstmusik** [comment]
I don't see anything in blue2.txt that would hint at a problem. You might try deleting the ~/.blue folder (or back it up) before starting up Blue.  Otherwise, I may have to install Ubuntu to diagnose this further. Alternatively, you might try installing Oracle JDK.

**Comment by kunstmusik** [comment]
Following up with this, I think this was a problem due to an issue in Csound itself, in particular the signalflowgraph opcodes.  I've gone back and forth with Michael Gogins about this issue and he's revised the code there and this should now be working with the latest Csound from the develop branch.  Rebuilding csound with latest from develop or removing libsignalflowgraph.so should address the problem.  I'll close this issue now; feel free to file a new issue if the problem continues.
